### PR TITLE
Stashing 1.1

### PR DESCRIPTION
**Pull request by ivdma**
Lot's of stashing happens nowadays...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -105,7 +105,7 @@ end
 ```
 
 ### Stashing PingPong balls
-Player is allowed to grab and stash any amount of PingPong balls in their hand. Player shamefully loses one life when caught stashing PingPong balls in pockets.
+Player is allowed to grab and stash any amount of PingPong balls in their hand. Player shamefully loses one life when caught stashing PingPong balls in pockets or hands.
```
Comment (martijnrusschen) [inline README.md]: any amount  -> More than one

**Review by martijnrusschen — CHANGES_REQUESTED** [review]

**Comment by ivdma** [comment]
I'd rather remove any specification of it

**Comment by martijnrusschen** [comment]
> Player is allowed to grab and stash PingPong balls in their hand. Player shamefully loses one life when caught stashing PingPong balls in pockets or hands.

The first sentence says you can grab balls in your hand, but the second sentence says you can't stash. Whut?

**Comment by martijnrusschen** [comment]
why not?

**Comment by Willianvdv** [comment]
@miraymazlumoglu where did you hide them?

**Comment by miraymazlumoglu** [comment]
You will never know

**Comment by miraymazlumoglu** [comment]
I will never show

**Comment by ivdma** [comment]
> You will never know
> 
> I will never show

That is why guilty person shamefully loses their life!

**Comment by ivdma** [comment]
🍃 🚢

**Pull request closed without merging**